src/main/java/org/terasology/launcher/gui/javafx/ApplicationController.java

```diff
@@ -411,6 +423,8 @@ public void update(final Path newLauncherDirectory, final Path newDownloadDirect
         }
 
         footerController.setHostServices(hostServices);
+
+        initializeComboBoxSelection();
```

Comment:
Having an *update* method call an *init* method sounds confusing. We should add list add a comment here why we need this, and probably a todo item (how) to replace this.